src/tests/electron/common/view-controllers/virtual-keyboard-view-controller.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { buildKeyboardButtonAutomationId } from 'electron/views/virtual-keyboard/virtual-keyboard-buttons';
+import { getAutomationIdSelector } from 'tests/common/get-automation-id-selector';
+import { SpectronAsyncClient } from 'tests/electron/common/view-controllers/spectron-async-client';
+import { ViewController } from './view-controller';
+
+export type VirtualKeyType = 'Up' | 'Down' | 'Left' | 'Right' | 'Enter' | 'Tab';
```

Comment:
This mirrors the `KeyEventCode` enum from adb-wrapper.ts. Would it be possible/worthwhile to link these together in a way that would make it easier to keep them in sync if we were to add additional values to the `KeyEventCode` enum?

Reply:
Thanks, I did not know about `KeyEventCode`! Refactored to use that instead of `VirtualKeyType`